tests/test_hsm.py

```diff
@@ -139,6 +139,24 @@ def test_moments_basic():
                 np.testing.assert_almost_equal(result.observed_shape.e2,
                                                distortion_2, err_msg = "- incorrect e2",
                                                decimal = decimal_shape)
+
+                # test for moments with a circular weight function
+                result_round = gal_image.FindAdaptiveMom(round_moments=True)
+                q = np.exp(-2.*math.atanh(total_shear))
+                theta = 0.5*math.atan2(g2, g1)
+                e_round = (1 - q**2)/(1 + q**2 + 2*q)
```

Comment:
Should say where this formula comes from: it's what we expect when integrating an elliptical Gaussian light profile with a round Gaussian weight function that has the same sigma to get the moments (can confirm analytically).